lib/system/comparisons.nim

```diff
@@ -251,9 +251,12 @@ proc max*[T](x: openArray[T]): T =
 
 proc clamp*[T](x, a, b: T): T =
   ## Limits the value `x` within the interval [a, b].
-  ## This proc is equivalent to but fatser than `max(a, min(b, x))`.
+  ## This proc is equivalent to but faster than `max(a, min(b, x))`.
   ## 
-  ## **Note:** `a <= b` is assumed and will not be checked.
+  ## .. warning:: `a <= b` is assumed and will not be checked (currently).
+  ##
+  ## **See also:**
+  ## `math.clamp` for a version that takes a `Slice[T]` instead.
```

Comment:
nit: maybe you use cross reference link like:
```
`clamp proc <math.html#clamp,T,Slice[T]>`_
```

Reply:
I've merged because this could be addressed in future PR, but more importantly because we should just implement https://github.com/nim-lang/RFCs/issues/125 and make life easier for everyone when writing links (including enabling generating warnings when a symbol reference is invalid)